app/backend/src/couchers/migrations/versions/71efe49ea92f_update_notification_enums_for_event_.py

```diff
@@ -0,0 +1,25 @@
+"""Update notification enums for event cancellation/deletion
+
+Revision ID: 71efe49ea92f
+Revises: 46e6b6a3d9ed
+Create Date: 2024-05-29 00:16:26.791019
+
+"""
+
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "71efe49ea92f"
+down_revision = "46e6b6a3d9ed"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    with op.get_context().autocommit_block():
```

Comment:
What does this one do?

Reply:
Altering a type has to be run outside of a transaction, which this context manager allows, if I understood that correctly. Read this in [Stackoverflow](https://stackoverflow.com/a/51439224) and on [alembic docs](https://alembic.sqlalchemy.org/en/latest/api/runtime.html#alembic.runtime.migration.MigrationContext.autocommit_block).

Howevery, I saw that you did it in a previous migration without it, so maybe it is not necessary...

Reply:
I undid this again to follow the existing migration style in the repo.